tools/buildstamp/get_workspace_status

```diff
@@ -14,6 +14,17 @@ set -e
 # If the script exits with non-zero code, it's considered as a failure
 # and the output will be discarded.
 
+# Provide an escape hatch for people building from github release tarballs,
+# but in non-Sorbet git repositories, to get somewhat more accurate version
+# information.
+if [ -f .github/SORBET_ARCHIVE_SHA ]; then
```

Comment:
I'm not particularly in love with putting our own config inside `.github/`. Out of curiosity, have you asked anyone on the build team about what their preferences are?

Reply:
I don't have strong opinions about where the file goes; putting it in `.github` seemed like a reasonable thing to do.  `.SORBET_ARCHIVE_SHA` is also a possibility.

I did not want to try to futz with setting an environment variable from Bazel's command-line and ensuring that it propagated into this script's execution, which is probably the best way to go about this.

Reply:
For posterity, `--{host_,}action_env` variables do not get propagated into this script's execution environment, so going through the filesystem seems like the only way forward.

The build team did not have strong opinions on this, given that it's cross-bazel invocation communication; they did note that they have done similar things in the past.